addons/binding/org.openhab.binding.tesla/ESH-INF/thing/models.xml

```diff
@@ -633,4 +635,14 @@
 		<label>Wake Up</label>
 		<description>Wake up the vehicle from a (deep) sleep</description>
 	</channel-type>
+    <channel-type id="allowwakeup" advanced="true">
```

Comment:
XML files should be indented using tabs according to our code formatter, currently its seem that it became a mix here?